api/views/__init__.py

```diff
@@ -343,6 +351,78 @@ def report_webcompat_issue(request):
     return response.Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
 
+class FirstForwardedEmailRateThrottle(throttling.UserRateThrottle):
+    rate = settings.FIRST_EMAIL_RATE_LIMIT
```

Comment:
Noting, when the user sends more then 5 requests in the frontned, need to handle it in the frontend @lloan .